packages/expo-dev-menu/app/views/DevMenuView.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { StyleSheet, PixelRatio, View } from 'react-native';
+
+import DevMenuContext, { Context } from '../DevMenuContext';
+import { StyledText } from '../components/Text';
+import Colors from '../constants/Colors';
+import DevMenuItemsList from './DevMenuItemsList';
+import DevMenuAppInfo from './DevMenuAppInfo';
+import { DevMenuAppInfoType, DevMenuItemAnyType } from '../DevMenuInternal';
+
+type Props = {
+  appInfo: DevMenuAppInfoType;
+  uuid: string;
+  devMenuItems: DevMenuItemAnyType[];
+  enableDevelopmentTools: boolean;
+  showOnboardingView: boolean;
+};
+
+class DevMenuView extends React.PureComponent<Props, undefined> {
+  static contextType = DevMenuContext;
+
+  context!: Context;
+
+  collapse = () => {
+    this.context?.collapse?.();
+  };
+
+  renderItems() {
+    return <DevMenuItemsList items={this.context.devMenuItems} />;
+  }
+
+  renderContent() {
+    const { appInfo } = this.props;
+
+    return (
+      <>
+        <DevMenuAppInfo appInfo={appInfo} />
+        <View style={styles.itemsContainer}>{this.renderItems()}</View>
+      </>
+    );
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+        {this.renderContent()}
+        {/* Enable this to test scrolling
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()}
+        {this.renderContent()} */}
```

Comment:
Let's be more clever and less copy-pasting 😉 
```suggestion
        {/* Enable this to test scrolling
        [...Array(10).keys()].map(this.renderContent)
    	 */}
```